src/screens/tezos-token-screen/tezos-token-screen.tsx

```diff
@@ -33,7 +33,7 @@ export const TezosTokenScreen = () => {
 
         <PublicKeyHashText publicKeyHash={selectedAccount.publicKeyHash} marginBottom={formatSize(16)} />
 
-        <HeaderCardActionButtons asset={TEZ_TOKEN_METADATA} />
+        <HeaderCardActionButtons asset={TEZ_TOKEN_METADATA} balance={tezosBalance} />
```

Comment:
What do you think about creating `useTezosTokenSelector` that will return `TokenInterface`?
(the same what we are doing in `SendModal` right now)
```
{
          ...emptyToken,
          ...TEZ_TOKEN_METADATA,
          balance: selectedAccount.tezosBalance.data
        }
```
And you will be able to pass a single prop instead.